ES_IDX_MANAGER_SETTINGS.yaml.template

```diff
@@ -1,13 +1,13 @@
 - name: elastic search human cell atlas
   endpoint: {endpoint}
   days: 7
-  index_format: '%Y.%m.%d'
+  index_format: '%Y-%m-%d'
   indices:
     - prefix: cwl
 
 - name: test configuration
   endpoint: {endpoint}
   days: 7
-  index_format: '%Y.%m.%d'
+  index_format: '%Y-%m-%d'
```

Comment:
Just curious: why the index format change?

Reply:
This is the format firehose uses when sending to ES for a one day configuration